notNeededPackages.json

```diff
@@ -2249,6 +2249,10 @@
             "libraryName": "jest-each",
             "asOfVersion": "24.3.0"
         },
+        "jest-expect-message": {
+            "libraryName": "jest-expect-message",
+            "asOfVersion": "1.1.0"
```

Comment:
CI fails, if I keep `1.0.4` here. They released server minor as well recently. So can stay like this as well.